jest.config.json

```diff
@@ -10,7 +10,7 @@
     "\\.svg\\?(component|source)": "<rootDir>/frontend/test/__mocks__/svgMock.jsx"
   },
   "transformIgnorePatterns": [
-    "<rootDir>/node_modules/(?!rehype-external-links/)"
+    "<rootDir>/node_modules/(?!(rehype-external-links|unist-util-visit|unist-util-visit-parents|space-separated-tokens|is-absolute-url)/)"
```

Comment:
packages were upgraded after removal of storybook from package.json, they all are in ESM, so we have to transpile them